.baseline/checkstyle/checkstyle-suppressions.xml

```diff
@@ -21,17 +25,5 @@
     <suppress files="\.(bdr|eot|gif|gzip|jar|json|otf|png|svg|ttf|woff|zip)$" checks="NewlineAtEndOfFile" />
 
     <!-- Generated code should not be subjected to checkstyle. -->
-    <suppress files="[/\\].*[/\\]generated_src[/\\]" checks="." />
-    <suppress files="[/\\].*[/\\]generated_testSrc[/\\]" checks="." />
-    <suppress files="[/\\].*[/\\]generated[/\\]" checks="." />
-
-    <!-- Certain AtlasDB projects are ignored. -->
-    <suppress files="atlasdb-client/*" checks="." />
-    <suppress files="atlasdb-commons/*" checks="." />
-    <suppress files="atlasdb-console/*" checks="." />
-    <suppress files="atlasdb-jdbc/*" checks="." />
-    <suppress files="atlasdb-jdbc-tests/*" checks="." />
-    <suppress files="cassandra-partitioner/*" checks="." />
-    <suppress files="commons-db/*" checks="." />
-    <suppress files="lock-impl/*" checks="." />
```

Comment:
if you put these into `custom-suppressions.xml` then they wouldn't be wiped out across upgrades